### PR TITLE
Support multiple params in tasks

### PR DESCRIPTION
**Pull request by crshnburn**
# Changes

Update how Gradle is invoked to support multiple parameters in the TASKS parameter

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of 
task changed or new task added)
- [x] Commit messages follow [commit message best 
practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [x] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this 
validation][validation]
  - [x] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [x] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [x] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [x] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [x] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

File: task/gradle/0.2/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Gradle
+
+This Task can be used to run a Gradle build on a gradle project.
+
+## Install the Task
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/gradle/0.1/gradle.yaml
```
Comment (vinamra28) [inline task/gradle/0.2/README.md:8]:
```suggestion
kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/gradle/0.2/gradle.yaml
```

File: task/gradle/0.2/gradle.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gradle
+  labels:
+    app.kubernetes.io/version: "0.1"
```
Comment (vinamra28) [inline task/gradle/0.2/gradle.yaml:6]:
```suggestion
    app.kubernetes.io/version: "0.2"
```

**Comment by vinamra28** [comment]
@crshnburn PR looks good. Can you please squash the commits and then we can merge this
/approve

**Comment by vinamra28** [comment]
> Catlin Output
> ```
> FILE: task/gradle/0.2/gradle.yaml
> WARN : Step "gradle-tasks" uses image "$(params.GRADLE_IMAGE)" that contains variables; skipping validation
> ```
> 
> Catlin script lint Output
> ```
> ERROR: /usr/bin/shellcheck, [-s sh] failed:
> 
> In gradle-gradle-tasks line 2:
> gradle $(params.TASKS)
>        ^-------------^ SC2046: Quote this to prevent word splitting.
> 
> For more information:
>   https://www.shellcheck.net/wiki/SC2046 -- Quote this to prevent word splitt...
> ```

@crshnburn needs to make catlin happy 😅

**Comment by crshnburn** [comment]
@vinamra28 Catlin has been made happy :)

**Pull request merged**